### PR TITLE
fix(css): for sticky value an inset property need to be set

### LINKED ISSUE #29704: Position: sticky description needs info about top, right, bottom, left requirement
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/position

### What specific section or headline is this issue about?

https://developer.mozilla.org/en-US/docs/Web/CSS/position#sticky

### What information was incorrect, unhelpful, or incomplete?

The sticky description doesn't note that at least one of top, right, bottom, or left, are required to be set.

### What did you expect to see?

The sticky description should include this string: "You must specify a threshold with at least one of top, right, bottom, or left for sticky positioning to behave as expected. Otherwise, it will be indistinguishable from relative positioning."

Currently that string is in the example for sticky positioning but it's important enough that it should be in the initial description.

https://developer.mozilla.org/en-US/docs/Web/CSS/position#sticky_positioning:~:text=You%20must%20specify%20a%20threshold%20with%20at%20least%20one%20of%20top%2C%20right%2C%20bottom%2C%20or%20left%20for%20sticky%20positioning%20to%20behave%20as%20expected.%20Otherwise%2C%20it%20will%20be%20indistinguishable%20from%20relative%20positioning.

### Do you have any supporting links, references, or citations?

No.

### Do you have anything more you want to share?

No.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/position`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/position
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/position/index.md
* Last commit: https://github.com/mdn/content/commit/4253659c02fcc4c9133f80e7a37f827c37b97045
* Document last modified: 2023-08-03T06:12:55.000Z

</details>

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- fixes https://github.com/mdn/content/issues/29704

### CODE REVIEW

File: files/en-us/web/css/position/index.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ position: unset;
 
     This value always creates a new [stacking context](/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context). Note that a sticky element "sticks" to its nearest ancestor that has a "scrolling mechanism" (created when `overflow` is `hidden`, `scroll`, `auto`, or `overlay`), even if that ancestor isn't the nearest actually scrolling ancestor.
 
+    > **Note:** At least one [inset](/en-US/docs/Web/CSS/inset) property on the axis in which the element needs to be kept in view needs to have non `auto` value. Otherwise, if both inset properties are `auto` on an axis then the `sticky` value behaves as `relative` on that axis.
```
Comment (estelle) [inline files/en-us/web/css/position/index.md:59]:
```suggestion
    > **Note:** At least one the [`inset`](/en-US/docs/Web/CSS/inset) shorthand property values of {{cssxref("top")}},  {{cssxref("right")}},   {{cssxref("bottom")}},  or {{cssxref("left")}} needs to be set to a none-`auto` value for the axis on which the element needs to stick. If both `inset` properties for an axis are set to `auto`, on that axis the `sticky` value will behave as `relative`.
```

**Review by estelle — COMMENTED** [review]
I found the sentence a little difficult to parse, and question whether the majority of CSS coders know `inset`, so made a suggestion. Feel free to edit my suggested edit ;)

File: files/en-us/web/css/position/index.md (hunk 1 above)
Comment (OnkarRuikar) [reply files/en-us/web/css/position/index.md]: For sake of completeness properties `inset-block-start`, `inset-inline-start`, etc. also need to be considered. I've tweaked your suggestion. Let me know [how it looks](https://github.com/mdn/content/pull/29737/commits/be2fea45a929edf311e7fbbde5dd9bbdca40549e).

**Review by estelle — APPROVED** [review]
Thank you! 🎉

**Pull request merged**